Database/Corrections/sodQuestFixes.lua

```diff
@@ -2358,6 +2358,33 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -424992,
             [questKeys.zoneOrSort] = sortKeys.ROGUE,
         },
+        [90139] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {{1161,1162,1163,1164}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.WARRIOR,
+            [questKeys.objectivesText] = {"Kill Troggs for the Skull-Shaped Geode. Then get hit by a Skullthumper's stun and use the geode to loot the rune."},
+            [questKeys.requiredSpell] = -425443,
+            [questKeys.zoneOrSort] = sortKeys.WARRIOR,
+        },
+        [90140] = {
+            [questKeys.name] = "Quick Strike",
+            [questKeys.startedBy] = {
+                ['objectStart'] = {408799},
+                ['creatureStart'] = {504,95,449,590,589,121,285,46,732,735,210487},
```

Comment:
Unsure if this is the correct syntax that we want to use, but as this quest requires two items dropped from multiple mobs to be brought to an object. This makes sense, but will require some assistance in case we use another way.

Reply:
See my comment below